### PR TITLE
Clarifications for 'Start Tasks'

### PR DESCRIPTION
**Pull request by svdarren**
When learning the Gitpod YML file, there were a few parts of the documentation that were unclear to me.  I'm proposing resolving two of them on this page.

### CODE REVIEW

File: src/docs/44_config_start_tasks.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ For instance, the start script for the Gitpod documentation repository is define
 
 ```yaml
 tasks:
-- command: mdbook serve --hostname 0.0.0.0
+- init: yarn && yarn build
```
Comment (svdarren) [inline src/docs/44_config_start_tasks.md]: The example cited here has been updated, and is actually more helpful to new users.  None of the examples showed two commands executing on the same terminal, so updating to the newer code is an easy to understand example.

File: src/docs/44_config_start_tasks.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ For instance, the start script for the Gitpod documentation repository is define
...
+  command: yarn dev --host 0.0.0.0
```
Comment (svdarren) [inline src/docs/44_config_start_tasks.md]: The example cited here has been updated, and is actually more helpful to new users.  None of the examples showed two commands executing on the same terminal, so updating to the newer code is an easy to understand example.

File: src/docs/44_config_start_tasks.md [hunk 3]
```diff
@@ -32,6 +33,14 @@ The script below will start a development-time web server in many npm projects:
...
   - command: npm run dev
 ```
+Task properties will control when a command is executed. Check the table below for an overview of the different starting scenarios.
+
+| Start Mode | Execution |
+| ---------  | -------   |
+| Fresh Workspace | `before && init && command` |
+| Restart Workspace | `before && command` |
+| Snapshot | `before && command` |
+| Prebuild | `before && init && prebuild` |
```
Comment (svdarren) [inline src/docs/44_config_start_tasks.md]: This table is really helpful, but prebuild was not included.  This is my understanding from the existing description, so please confirm this is correct before approving.

File: src/docs/44_config_start_tasks.md [hunk 4]
```diff
@@ -59,14 +68,6 @@ tasks:
...
 ```
 
-Check the table below for an overview of the different starting scenarios.
-
-| Start Mode | Execution |
-| ---------  | -------   |
-| Fresh Workspace | `before && init && command` |
-| Restart Workspace | `before && command` |
-| Snapshot | `before && command` |
-
```
Comment (svdarren) [inline src/docs/44_config_start_tasks.md]: This table is really helpful for new users, so I'm suggesting it should be moved earlier in the document.

**Comment by JesterOrNot** [comment]
Looks good!

**Pull request merged**